src/App.js

```diff
@@ -2,21 +2,26 @@ import './App.css'
 import { Route, Routes } from 'react-router-dom'
 import Header from './components/Header/Header'
 import Nav from './components/Nav/Navbar'
-import SearchBar from './components/SearchBar/SearchBar'
+// import SearchBar from './components/SearchBar/SearchBar'
 import ReviewList from './components/Reviews/ReviewList'
 import ReviewSingle from './components/Reviews/ReviewSingle'
 import Footer from './components/Footer/Footer'
+import ReviewComments from './components/Comments/ReviewComments'
 
 function App() {
   return (
     <section className="App">
       <Header />
       <Nav />
-      <SearchBar />
+      {/* <SearchBar /> */}
       <Routes>
         <Route path="/" element={<ReviewList />} />
         <Route path="/reviews" element={<ReviewList />} />
         <Route path="/reviews/:review_id" element={<ReviewSingle />} />
+        <Route
+          path="/reviews/:review_id/comments"
+          element={<ReviewComments />}
+        />
```

Comment:
You're showing the comments twice which is unnecessary. Since they're being shown on <ReviewSingle /> there's no need for a whole separate route and page. Or if you prefer having a separate comments page, then you don't have to put the comments on the review single page.